FAQ/en.md

```diff
@@ -0,0 +1,73 @@
+# Frequently Asked Questions
+
+## General
+
+### What is Quaver?
+
+Quaver is a community-driven rhythm game, aiming to bring VSGRs to the next level. The game is open to players of all skill levels, casual and seasoned.
+
+### How is Quaver different from other games?
+
+The game is fully open-source, allowing anyone to help and improve the game. Furthermore, we are aiming to provide a new VSGR experience by bringing competitive concepts, such as competitive matchmaking.
```

Comment:
The game is fully open-source, which allows*

VSRG*

Reply:
applied and oops